src/main.py

```diff
@@ -54,6 +51,7 @@ def setup(self):
 
         self.arena = load_arena_by_name("default")
         self.arena.init_for_drawing()
+        self.sprite_lists.wall_sprite_list = self.arena.wall_sprite_list
```

Comment:
Should this be inverted, where the arena's `init_for_drawing` is instead given a reference to `sprite_lists`?

Reply:
Yes, that's exactly what I wanted to do but wasn't sure if it was alright to dig into that class with this logic. With your blessing, I'll pass arena `sprite_lists.players` in it's constructor and have it do that. Is it cool if I have SpriteLists draw that list as well?